stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/HiddenMessageListItemFilter.kt

```diff
@@ -0,0 +1,24 @@
+package io.getstream.chat.ui.sample.feature.chat
+
+import com.getstream.sdk.chat.adapter.MessageListItem
+import io.getstream.chat.android.ui.messages.view.MessageListView
+import io.getstream.chat.android.ui.utils.extensions.isDeleted
+import io.getstream.chat.android.ui.utils.extensions.isGiphyEphemeral
+
+object HiddenMessageListItemFilter : MessageListView.MessageListItemFilter {
```

Comment:
Shouldn't this be the default implementation that we ship in the library, instead of being part of the sample app? Feels like it's tied to the behaviour of the ViewHolders that we _do_ ship.

Reply:
I'm not sure. For example in the old sample app, we showed their deleted messages intentionally.

Reply:
This is a separate `MessageListView` class, so it wouldn't impact the old sample.

Reply:
Hmm... Makes sense. Will fix it